couchbase-entity/src/main/java/com/kaufland/Logger.kt

```diff
@@ -41,4 +41,15 @@ class Logger internal constructor(processingEnv: ProcessingEnvironment) {
     fun hasErrors(): Boolean {
         return hasErrors
     }
+
+    fun abortWithError(e: PostValidationException) {
+        if(e.causingElements.isNotEmpty()){
```

Comment:
Formatting, a ktlint format would maybe be good for the project :) But in a separate PR.